### ISSUE TITLE
Git Error while pre-commit 

### ISSUE THREAD

**Issue by mpasternak** [ForkIssues/Tracker#1660]
Hi,

``pre-commit`` is in my path, installed with ``pipx``, so it is a user-wide installation. Every single shell I run has access to this files in ``PATH``. 

```
⇒  pipx  list|grep pre-commit
   package pre-commit 2.19.0, installed using Python 3.9.12
    - pre-commit
    - pre-commit-validate-config
    - pre-commit-validate-manifest
```

Even if available in PATH, Fork is somehow not able to run that. How do I fix that? 

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/598469/181825018-db8df9c9-7b04-4867-ad7d-6c81e9e21394.png">

**Comment by DanPristupov** [comment]
Try to run fork from CLI using `open -a Fork` command.
You can find more info here https://github.com/ForkIssues/Tracker/issues/430#issuecomment-429013890

**Comment by mpasternak** [comment]
How about Fork tries to launch subprocesses with proper environment?

**Comment by DanPristupov** [comment]
Which environment is proper? In unix ENV belongs to process, not to the system.

**Comment by mpasternak** [comment]
Let me rephrase. 

How about an option to launch it via shell so it gains all the required environment variables?

**Comment by DanPristupov** [comment]
> How about an option to launch it via shell so it gains all the required environment variables?

What do you mean? `open -a Fork` does exactly that.

**Comment by mpasternak** [comment]
I paid for Fork to be able to click few buttons and have it done. 

Currently the application doesn’t do what I bought it for so I guess I’ll go somewhere else. Thanks. 

Wysłane z iPhone'a

**Comment by DanPristupov** [comment]
I made the refund. The money will be back to your account soon.

**Comment by mpasternak** [comment]
Well I kindly request you reopen this issue as the problem wasn’t solved.
